include/awkward/cpu-kernels/getitem.h

```diff
@@ -4,6 +4,7 @@
 #define AWKWARDCPU_GETITEM_H_
 
 #include "awkward/cpu-kernels/util.h"
+#include <memory>
```

Comment:
Now that the `std::shared_ptrs` have been removed, I don't think you need to include `memory` anymore.